packages/fast-usdc/src/exos/transaction-feed.js

```diff
@@ -73,45 +75,94 @@ export const prepareTransactionFeedKit = (zone, zcf) => {
          * @returns {Promise<Invitation<OperatorKit>>}
          */
         makeOperatorInvitation(operatorId) {
-          const { admin } = this.facets;
+          const { creator } = this.facets;
           trace('makeOperatorInvitation', operatorId);
 
           return zcf.makeInvitation(
             /** @type {OfferHandler<OperatorKit>} */
             seat => {
               seat.exit();
-              return admin.initOperator(operatorId);
+              return creator.initOperator(operatorId);
             },
             INVITATION_MAKERS_DESC,
           );
         },
+        /** @param {string} operatorId */
+        async initOperator(operatorId) {
+          const { operators, pending } = this.state;
+          trace('initOperator', operatorId);
+
+          const operatorKit = makeOperatorKit(
+            operatorId,
+            this.facets.operatorPowers,
+          );
+          operators.init(operatorId, operatorKit);
+          pending.init(operatorId, makeScalarBigMapStore('pending evidence'));
+
+          return operatorKit;
+        },
+
         /** @param {string} operatorId */
         async removeOperator(operatorId) {
-          const { operators: oracles } = this.state;
+          const { operators } = this.state;
           trace('removeOperator', operatorId);
-          const kit = oracles.get(operatorId);
-          kit.admin.disable();
-          oracles.delete(operatorId);
+          const operatorKit = operators.get(operatorId);
+          operatorKit.admin.disable();
+          operators.delete(operatorId);
         },
       },
+      operatorPowers: {
+        /**
+         * Add evidence from an operator.
+         *
+         * @param {CctpTxEvidence } evidence
+         * @param {OperatorKit} operatorKit
+         */
+        submitEvidence(evidence, operatorKit) {
+          const { pending } = this.state;
+          trace(
+            'submitEvidence',
+            operatorKit.operator.getStatus().operatorId,
+            evidence,
+          );
+          const { operatorId } = operatorKit.operator.getStatus();
 
-      admin: {
-        /** @param {string} operatorId */
-        async initOperator(operatorId) {
-          const { operators: oracles } = this.state;
-          trace('initOperator', operatorId);
+          // TODO verify that the operator is one made by this exo
+          // This doesn't work...
+          // operatorKit === operators.get(operatorId) ||
+          //   Fail`operatorKit mismatch`;
 
-          const oracleKit = makeOperatorKit(operatorId);
-          oracles.init(operatorId, oracleKit);
+          // TODO validate that it's valid before accepting
```

Comment:
just curious... we can come back to the TODO later...

The exo guard doesn't suffice? There are things that match `CctpTxEvidenceShape` that are not valid?

Reply:
> match CctpTxEvidenceShape that are not valid

Yes, such as if the `recipientAddress` is not the settlement account or the EUD is not a chain supported by FU. I'l add that in the comment.